packages/i18n/package.json

```diff
@@ -10,9 +10,11 @@
   },
   "main": "./src/index.js",
   "type": "module",
+  "sideEffects": false,
```

Comment:
Why is this needed?

Reply:
This helps webpack to optimize imports, see https://webpack.js.org/configuration/optimization/#optimizationsideeffects